pkg/operator/status.go

```diff
@@ -318,6 +332,14 @@ func (c *Controller) computeStatus(tuned *tunedv1.Tuned, conditions []configv1.C
 			degradedCondition.Message = fmt.Sprintf("%v/%v Profiles with bootcmdline conflict", numConflict, len(profileList))
 		}
 
+		numMCLabelsAcrossMCP := c.numMCLabelsAcrossMCP(profileList)
```

Comment:
Will this report a degraded condition for all profiles when any one reports a conflict?